boards/imixv2/apps/sensors/Makefile

```diff
@@ -0,0 +1,16 @@
+# Makefile for user application
```

Comment:
Do we need this sensors app? The one in examples should sample all sensors, and only the sensors actually on a platform.